frontend-react/src/content/home/index.mdx

```diff
@@ -1,4 +1,9 @@
 ---
+meta:
+    openGraph:
+        image:
+            src: /assets/img/opengraph/reportstream-tagline.png
+            altText: ""
```

Comment:
nit: is it worth leaving this blank so the default alt text appears? probably better than have a blank string for the html attribute.

Reply:
I've fixed the createMeta function to specially-handle cases where the right-side is a blank string (and returns left-side instead) so that this pattern won't break expectations.